### PR TITLE
Add bluemix/cloud-foundry support

### PR DESCRIPTION
**Pull request by kraman**
Connect to strongloop-internal/scrum-nodeops#876

### CODE REVIEW

File: test/test-bluemix.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    ]
+  });
+
+  t.test('Wrapper parses addon-info', function(t) {
+    runner(MockWebsocketChannelModule, function() {
+      t.equal(
+        MockWebsocketChannelModule.ch.registrationUrl,
+        'https://bluemix.do.strongloop.com/executor-control'
+      );
+      t.deepEqual(
```
Comment (rmg) [inline test/test-bluemix.js]: Why not `t.equal()`? Is this meant to be comparing more than just a string?

**Comment by rmg** [comment]
Minor nit in the tests, otherwise LGTM.

**Comment by kraman** [comment]
@rmg please take another look. I shuffled around some code.

**Comment by rmg** [comment]
I think the readability could be improved by extracting the registration out to a separate function. Other than that, LGTM.

**Pull request merged**